lte/gateway/c/session_manager/GrpcMagmaUtils.cpp

```diff
@@ -79,3 +79,9 @@ std::string indentText(std::string basicString, int indent) {
   }
   return result;
 }
+
+void prepend_imsi_with_prefix(std::string& imsi) {
```

Comment:
If the input is to be modified, let's pass a pointer and not a reference.

Reply:
Changed the function prototype to `std::string prepend_imsi_with_prefix(const std::string& imsi);` , so it takes input parameter and checks if the "IMSI" substring is present or not, if it isn't present, it returns `"IMSI" + imsi` else it simply returns `imsi`